### ISSUE TITLE
D-Pad Up Disables Gyro.

### ISSUE THREAD

**Issue by Dext1** [JibbSmart/JoyShockMapper#20]
There seems to be a hidden binding left in the code where gyro is disabled when up on the d-pad is held. No other button does this.

**Comment by JibbSmart** [comment]
Right. Might be something to do with d-pad up being the 0th input. I'll have a look. Thanks.
